gcp/modules/couchdb/main.tf

```diff
@@ -3,13 +3,47 @@ terraform {
 }
 
 variable "values_dir" {}
+variable "release_namespace" {
+	default = "gpii"
+}
 
 module "couchdb" {
   source = "/exekube-modules/helm-template-release"
 
   release_name      = "couchdb"
-  release_namespace = "gpii"
+  release_namespace = "${var.release_namespace}"
   release_values    = "${var.values_dir}/couchdb.yaml"
 
   chart_name = "../../../../../charts/couchdb"
 }
+
+variable "couchdb_admin_username" {}
+variable "couchdb_admin_password" {}
+
+resource "null_resource" "couchdb_finish_cluster" {
+  depends_on = ["module.couchdb"]
+
+  provisioner "local-exec" {
+    command = <<EOF
+      while [ "$PODS_READY" != "true" ]; do
+        PODS_READY=true
+        echo Waiting for all CouchDB pods to become Running...
+        for status in $(kubectl get pods --namespace ${var.release_namespace} -l app=couchdb -o jsonpath='{.items[*].status.phase}'); do
+          if [ "$status" != "Running" ]; then
+            PODS_READY=false
+          fi
+        done
+        sleep 5;
+      done
+      while [ "$STATUS" != "\"Cluster is already finished\"" ]; do
```

Comment:
Please single quote the string (including the double quotes), rather than double quoting and then escaping double quotes inside.

Reply:
Done